src/backend/auth/admin_access_endpoints.py

```
@@ -0,0 +1,137 @@
+@admin_access_router.post("/users/{user_id}/reactivate")
+def reactivate(user_id: str, request: Request, admin: dict = Depends(require_admin)):
+    return _transition(user_id, "active", bump=False, admin=admin, request=request)
+
+
+class _CreateOrg(BaseModel):
+    name: str
+    owner_user_id: str
+
+
+@admin_access_router.post("/orgs", status_code=201)
+def create_org(body: _CreateOrg, request: Request, admin: dict = Depends(require_admin)):
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)